aten/src/ATen/core/dispatch/OperatorEntry.h

```diff
@@ -102,6 +105,8 @@ class CAFFE2_API OperatorEntry final {
 
   OperatorName name_;
   c10::optional<FunctionSchema> schema_;
+  c10::optional<std::string> debug_;
+  // INVARIANT: schema_.has_value() == debug_.has_value()
```

Comment:
should we explicitly model this instead by having a `c10::optional<SchemaInfo>` with

```
struct SchemaInfo final {
    FunctionSchema schema;
    std::string debug;
}
```

That would make the invariant be enforced by the compiler instead of having to rely on the comment here.

Reply:
I considered it, but adding another struct seemed annoying. Up to you.